lib/services/common-api-presenter.js

```diff
@@ -39,7 +40,8 @@ function CommonApiPresenterFactory(
     lookupService,
     profiles,
     templates,
-    _
+    _,
+    Env
```

Comment:
Since `Env` is an instance (https://github.com/RackHD/on-core/blob/master/lib/services/environment.js#L82), so as convention, its name should begin with lowercase. (if it is a Class, it should begin with uppercase) So change it from `Env` to `env`